### PR TITLE
Update QUEST_JOBSTEP.tsv

### PR DESCRIPTION
**Pull request by Dastev**
Correcting the word "Keepsakes" which has been translated as "Keepstakes".  Also fixed "deads" to "dead" since dead doesn't have a plural form and got one or two small translation improvements.  Will do more later!

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -3112,9 +3112,9 @@ QUEST_JOBSTEP_20150803_003111	I don't have guts for it
 QUEST_JOBSTEP_20150803_003112	Received enough balm. Restore the Vakarine Statue in Fedimian and Ausrine Statue in Klaipeda.
 QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in the west forest of Siaulai, the Necromancer Master in Mulus Corridor, and the Chronomancer Master in Stele Road.
 QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Go back to the Sorcerer Master.
-QUEST_JOBSTEP_20150803_003115	Obtain the keepstakes which you could feel strong regrets
-QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepstake with stronger grudge. Obtain the keepstakes that you can feel strong regrets from Bidentis Shrine.
-QUEST_JOBSTEP_20150803_003117	You've collected the keepstakes with strong regrets. Go back to the Necromancer Master.
+QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes which you could feel strong regrets
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:3115]: This sounds off. As if it's missing a "from" at the end. Also, period at the end of the sentence?

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -3112,9 +3112,9 @@ QUEST_JOBSTEP_20150803_003111	I don't have guts for it
...
+QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepsake with a stronger grudge. Obtain the keepsakes that you can feel strong regrets from Bidentis Shrine.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:3116]: Says "stronger grudge" vs "stronger regret" in other lines - should be unified.

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -2982,7 +2982,7 @@ QUEST_JOBSTEP_20150729_002981	If you had explored the regions completely in the
 QUEST_JOBSTEP_20150729_002982	It's been recorded completely.{nl}Good. I will pass this to the Alchemist Master.{nl}Now is my turn to help you.
 QUEST_JOBSTEP_20150729_002983	The power of Necromancer will always be expressed through the materials that act as catalysts.{nl}That power will be influenced by the abilities of Necromancer himself and his power.{nl}If you want to advance more as a Necromancer, it's important that you obtain those materials. 
 QUEST_JOBSTEP_20150729_002984	Strong grudge is swirling around Bidentis Shrine according to what I found out.{nl}When you go there, you will find the materials that possess the thoughts of the dead ones.{nl}If you could obtain those, I will open the door of a higher level Necromancer.
-QUEST_JOBSTEP_20150729_002985	The keepstakes express stronger power that belong to the spirits with regret in their past lives.{nl}Find the keepstakes that you could feel those regrets.
+QUEST_JOBSTEP_20150729_002985	The keepsakes express stronger power that belong to the spirits with regret in their past lives.{nl}Find the keepsakes that you could feel those regrets.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:2985]: This one sounds odd, as well. Again, regret vs. grudge.

**Comment by Soukyuu** [comment]
Btw, as IMC decided in #900, you now have to put a $ in front of the translation. So for example
`QUEST_JOBSTEP_20150729_002985  $The keepsakes express stronger power that belong to the spirits with regret in their past lives.{nl}Find the keepsakes that you could feel those regrets.`

File: QUEST_JOBSTEP.tsv (hunk 1 above)
Comment (Dastev) [inline QUEST_JOBSTEP.tsv:3115]: I agree with all you've said!  I'm kind of new to this GitHub thing and I was only looking at the keepstake and not at the sentences as a whole heh.  I'll check them back and fix them up.  Just the typo that really caught my eye.  Also thanks on the $ hint, I had no clue!

**Comment by emailboxu** [comment]
FYI Find -> Replace All tends to screw with other people's pull requests so if you could instead manually edit lines X~Y like everyone else is doing and put which lines you edited in your title it would be a lot better. Also it ignores the other potential grammatical/spelling errors in a line because you haven't actually checked them yourself.

It should only be used if a term is being switched completely that imc has given clearance for.

**Comment by Soukyuu** [comment]
I agree. What I did on one PR was to do a replace all, but also edit each sentence where I replaced stuff.

**Comment by Dastev** [comment]
Ahh I see.  Sure thing!  I'll make sure to check the entire sentences when using find.  I -actually- just used find and then replaced the words manually, but I admitedly didn't check the rest of the lines :p  

Is there any way for me to erase/edit this pull request?  Kinda new here on Github.

**Comment by Soukyuu** [comment]
You don't have to. Simply fix what is wrong and push another commit on the same branch. It will land in this PR automatically.

**Comment by Dastev** [comment]
Got it.  Much obliged.

**Comment by terrapin47** [comment]
For reference, this old PR edited similar parts of this file.  Might help when making changes.
https://github.com/Treeofsavior/EnglishTranslation/pull/823
I'm unsure if the original contributor will be returning.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**